src/docs/cli/releases.mdx

```diff
@@ -13,7 +13,7 @@ Because releases work on projects you will need to specify the organization and
 
 ## Creating Releases
 
-Releases are created with the `sentry-cli releases new` command. It takes at the very least a version identifier that uniquely identifies the releases. There are a few restrictions -- the release name cannot contain newlines, spaces, or "\\\", be ".", "..", or exceed 200 characters. The value can be arbitrary, but for certain platforms, recommendations exist:
+Releases are created with the `sentry-cli releases new` command. It takes at the very least a version identifier that uniquely identifies the releases. There are a few restrictions -- the release name cannot contain newlines, spaces, "/" or "\\\", be ".", "..", or exceed 200 characters. The value can be arbitrary, but for certain platforms, recommendations exist:
```

Comment:
see the following edit for a suggested breakout of items not allowed in the name